cime_config/allactive/config_compsets.xml

```diff
@@ -320,17 +320,27 @@
 
 <compset>
   <alias>CRYO1850</alias>
-  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
 <compset>
   <alias>CRYO1850-4xCO2</alias>
-  <lname>1850SOI_EAM%CMIP6-4xCO2_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1850SOI_EAM%CMIP6-4xCO2_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
 <compset>
   <alias>CRYO1950</alias>
-  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>CRYO1850-DISMF</alias>
+  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBDISMF_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>CRYO1950-DISMF</alias>
+  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBDISMF_MOSART_SGLC_SWAV</lname>
 </compset>
```

Comment:
@darincomeau and @jonbob, there would be no value in defining `WCYCL1850-DISMF` and `WCYCL1950-DISMF` with identical `lname`s to these, right?  That is, there isn't any way the alias is used internally in the code to change namelist options or anything like that that isn't already in the long name, right?  I'm 90% sure that's the case but wanted to check.

Reply:
@darincomeau and @jonbob, in particular what I'm wondering about is where the difference between Cryo's and WC's GM parameters get set.  Are these tied to (P)ISMF being on?

Reply:
@xylar -- some other component's settings go off the date prefix

Reply:
@xylar - right now all the GM parameter settings are based on the ocn_grid and not tied to (P)ISMF. However, if we're trying to move to all ocean grids having ice shelf cavities by default, then we'll have to implement the different setting based on something besides grid name

Reply:
a bit late here but I agree there wouldn't be value in defining `WCYCL1850-DISMF`; since we don't have a data iceberg file for the non-wISC meshes (as of now), then that compset paired with non-wISC grids would be pretty wacky wrt to AIS runoff (snowcapping disabled, but only partly prescribed).

This brings up a point that we should double-check that indeed AIS snowcapping is disabled in these `DISMF` compsets. I suspect it is, but an easy way to check is that the `removedIceRunoff`, `removedRiverRunoff` variables in the files output by the ocean conservation analysis member (now on by default for any `wISC` grids) are non-zero in runs with these compsets.

Reply:
Thanks @darincomeau, that's helpful.

What I would like a `WCYCL1850-DISMF` to be is the same namelist options as a `WCYCL1850` but using the ECwISC mesh, DIB and DISMF.  Apparently, we would have to think about how to specify this.  I don't think the approach of trying certain parameter choices that are more appropriate for science campaigns instead to specific meshes will work going forward.  As @jonbob says, we will need to figure out another way (e.g. another flag in the long name) to control different namelist options for the same mesh.

Reply:
>What I would like a WCYCL1850-DISMF to be is the same namelist options as a WCYCL1850 but using the ECwISC mesh, DIB and DISMF. Apparently, we would have to think about how to specify this.

Yeah, in general the compset is largely agnostic of the grid, and vice-versa. The (exception) analogy would be how currently the `CRYO` compsets can only be run with `wISC` meshes, and users would get an error in trying to build/run a `CRYO1850.ne30pg2_EC30to60E2r2` case, for example. But we don't have a way to stop the user from trying to do that.

Reply:
Right, but because we've chosen to tie GM parameters (like N2 vs. constant) to meshes rather than compsets, there currently isn't a way to get the WCYCL compset with the ECwISC mesh without also getting the CRYO (rather than the WCYCL) GM coefficients.

Reply:
Yes that's by design since so many models can run at multiple resolutions.  A case is a compset and a resolution.  CIME doesn't contain any internal logic to capture invalid compset-resolution combinations.  That logic could be put in the buildnml for the model.

Reply:
Yes, but I think there are some config options that need to be tied to compaset *and* grids, and are currently only tied to grids.  We'll need to chat about this tomorrow at the MPAS DevOps meeting.